fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -94,4 +134,66 @@ public void setFedoraData(final Path fedoraData) {
         this.fedoraData = fedoraData;
     }
 
+    /**
+     * @return The JMS host
+     */
+    public String getJmsHost() {
```

Comment:
Of all of the following "getters", only `getVelocityLog()` appears to be used. Should the others be removed?

Reply:
I believe all of them are referenced in SpEL ( i.e. "#{fcrepoPropsConfig.member}" references in other parts of the codebase.  So not recommended.